src/components/atoms/Input/InputElement.module.css

```diff
@@ -100,6 +100,12 @@
   padding-left: 0.5rem;
 }
 
+.algorithmLabel {
```

Comment:
can't quickly see how we use this but this seems too specific to be put into our generic input styles, and should probably live somewhere else